components/image-controller/development/kustomization.yaml

```diff
@@ -2,12 +2,12 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 resources:
 - ../base
-- https://github.com/redhat-appstudio/image-controller/config/default?ref=e5a29db5772c85b84d3246597e5b39f229d2925a
+- https://github.com/redhat-appstudio/image-controller/config/default?ref=0f0b61cef9f667bb8a93d79fc03554fff9bb335b
 
 images:
 - name: quay.io/redhat-appstudio/image-controller
   newName: quay.io/redhat-appstudio/image-controller
```

Comment:
@mshaposhnik  the image ref probably needs to be updated according to [this](https://github.com/redhat-appstudio/image-controller/blob/0f0b61cef9f667bb8a93d79fc03554fff9bb335b/.tekton/image-controller-push.yaml#L25)

Reply:
I think we want to use released image on stage, so it shouldn't be changed.